docker/usr/local/package/espeak.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Exit immediately on error
+set -e
+
+# Get our method as the name of this script and the url
+METHOD=$(basename "$0")
+URL="$1"
+shift
+
+# Set installation prefix, default to /usr/local unless an external power says otherwise
+PREFIX=${PREFIX:-"/usr/local"}
+
+BUILD_FOLDER="/var/tmp/build"
+
+# Dont pull in the toolchain arguments as they seem to break espeaks build
+# . "${PREFIX}/toolchain.sh"
```

Comment:
Remove commented code

Reply:
I was going to, but I kept it in as otherwise comment on the line above has nothing to comment!